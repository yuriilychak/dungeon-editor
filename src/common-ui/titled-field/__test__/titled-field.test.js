import React from "react";
import { createMount } from "@material-ui/core/test-utils";
import TitledField from "../titled-field";

describe("TitledField test",()=> {
    it('default snapshot', () => {
        const wrapper = createMount()(
            <TitledField
                title="test"
                titleWidth={100}
            >
                Test
            </TitledField>
        );
        expect(wrapper.html()).toMatchSnapshot();
    });
});
