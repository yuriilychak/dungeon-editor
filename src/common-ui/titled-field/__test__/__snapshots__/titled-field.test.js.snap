// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitledField test default snapshot 1`] = `"<div class=\\"titled-field-root\\"><div class=\\"titled-field-title\\" style=\\"width: 100px;\\">test:</div><div class=\\"titled-field-body\\">Test</div></div>"`;
